frigate/config.py

```diff
@@ -388,6 +388,7 @@ class BirdseyeConfig(FrigateBaseModel):
 # uses BaseModel because some global attributes are not available at the camera level
 class BirdseyeCameraConfig(BaseModel):
     enabled: bool = Field(default=True, title="Enable birdseye view for camera.")
+    order: int = Field(default=0, title="Position of the camera in the birdseye view.")
```

Comment:
Please add this option to the documentation.

Reply:
Thank you for raising that, a very good point.
The docs are awesome for Frigate 💯